src/pages/loginphone.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react'
+import { NextPage } from 'next'
+
+import EnterPasswordPanel from '@/components/Login/EnterPasswordPanel'
+import EnterPhonePanel from '@/components/Login/EnterPhonePanel'
+import LoginHeader from '@/components/Login/LoginHeader'
+import { withPublic } from '@/components/PrivateRoute'
+
+const Login: NextPage = () => {
+  const pageTitle = 'Phone Login'
+  const primaryMessage = 'Enter your phone number'
+  const secondaryMessage = 'Please enter your phone number'
```

Comment:
The primary and secondary message both say the same thing, do we need both of them?

Reply:
Removed 'secondaryMessage' from
- src/pages/loginphone.tsx
- src/components/Login/LoginHeader.tsx